src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -171,72 +187,123 @@ const MovingAverageAggUi = props => {
             />
           </EuiFormRow>
         </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow
-            id={htmlId('minimize')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.minimizeLabel"
-              defaultMessage="Minimize"
-            />)}
-          >
-            <EuiComboBox
-              placeholder={intl.formatMessage({ id: 'tsvb.movingAverage.minimize.selectPlaceholder', defaultMessage: 'Select' })}
-              options={minimizeOptions}
-              selectedOptions={selectedMinimizeOption ? [selectedMinimizeOption] : []}
-              onChange={handleSelectChange('minimize')}
-              singleSelection={{ asPlainText: true }}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow
-            id={htmlId('predict')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.predictLabel"
-              defaultMessage="Predict"
-            />)}
-          >
-            {/*
-              EUITODO: The following input couldn't be converted to EUI because of type mis-match.
-              Should it be text or number?
-            */}
-            <input
-              className="tvbAgg__input"
-              type="text"
-              onChange={handleNumberChange('predict')}
-              value={model.predict}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
       </EuiFlexGroup>
 
-      <EuiSpacer size="m" />
+      {
+        (
+          model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL ||
```

Comment:
I see that we have duplicated block: 

```
model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL ||
model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_DOUBLE ||
model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE

```
What about to create a variable e.g. isExponentalType?

Reply:
Nice catch! The extra check has been removed.